### DISCUSSION TITLE
packageRules matchGroupName

### DISCUSSION THREAD

**Discussion by igas** [renovatebot/renovate#28553] [Request Help]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Mend Renovate hosted app on github.com

### Please tell us more about your question or problem

Hi, I was looking through the docs, but could not find anything. I would like to match on a group name. Specifically, I want to rely on upstream preset, like `group:allDigest` and add a rule to it. In my case `automerge`. It might be more of the feature request, or it can be done already.

Thanks.

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment]
We have already been planning to implement `matchBranchNames`, would that be a suitable alternative?

**Reply by igas** [reply]
I think it'll be definitely solving a problem, although it still feels like naming can be accidentally changed. Maybe it could be done in more of the `extends` way. I have not tried, but I think currently it won't work, as they are on different levels.

Something like:

```json
{
  "packageRules": [
    {"extends": "group:allDigest", "automerge": true}
  ]
}
```
